blockchain/indexers/indexers_test.go

```diff
@@ -610,7 +610,7 @@ func TestProveUtxos(t *testing.T) {
 		uView := csnChain.GetUtreexoView()
 		err = uView.VerifyAccProof(proof.HashesProven, proof.AccProof)
 		if err != nil {
-			t.Fatal(fmt.Sprintf("TestProveUtxos fail. Failed to verify proof err: %v", err))
+			t.Fatalf(fmt.Sprintf("TestProveUtxos fail. Failed to verify proof err: %v", err))
```

Comment:
`s/t.Fatalf(fmt.Sprintf("TestProveUtxos fail. Failed to verify proof err: %v", err))/t.Fatalf("TestProveUtxos fail. Failed to verify proof err: %v", err))`